server/src/routes/api/v1/movieReviewRouter.js

```diff
@@ -8,15 +8,17 @@ const movieReviewRouter = new express.Router({ mergeParams: true });
 
 movieReviewRouter.post("/", async (req, res) => {
   const movieId = req.params.id;
+  const userId = req.user.id;
   const { body } = req;
   const formInput = cleanUserInput(body);
-  const { content, userId } = formInput;
+  const { content, rating } = formInput;
 
   try {
     const newReview = await Review.query().insertAndFetch({
       movieId,
       userId,
       content,
+      rating: parseInt(rating),
```

Comment:
we don't need to parse the string rating, postgres will type coerce for us so long as we tell jsonSchema that rating can be string or integer. The type coercion means that even if we provide a string, postgres will save the value as an integer for us